docs/difficulty_tests/index.rst

```diff
@@ -0,0 +1,50 @@
+.. _difficulty_tests:
+
+################################################################################
+Difficulty Tests
```

Comment:
this tests are valid. but we have to add that it is not maintained and not being executed on hive